doc/nrf/ug_thread_tools.rst

```diff
@@ -472,39 +441,25 @@ Running the OpenThread POSIX applications
 
 Use the following radio URL parameter to connect to an RCP node.
 
-* For UART transport:
-
-  .. parsed-literal::
-     :class: highlight
-
-     'spinel+hdlc+uart://\ *ncp_uart_device*\ ?uart-baudrate=\ *baud_rate*'
-
-  Replace the following parameters:
-
-    * *ncp_uart_device* - Specifies the location of the device, for example: :file:`/dev/ttyACM0`
-    * *baud_rate* - Specifies the baud rate to use.
-      The Thread Co-Processor sample supports baud rate ``1000000``.
-
-  For example, to use ``ot-daemon`` with UART transport, enter the following commands::
+.. code-block:: console
 
-     sudo ./build/posix/src/posix/ot-daemon 'spinel+hdlc+uart:///dev/ttyACM0?uart-baudrate=1000000' --verbose
-     sudo ./build/posix/src/posix/ot-ctl
+   'spinel+hdlc+uart://\ *ncp_uart_device*\ ?uart-baudrate=\ *baud_rate*'
 
-* For USB transport:
+Replace the following parameters:
 
-  .. parsed-literal::
-     :class: highlight
+   * *ncp_uart_device* - Specifies the location of the device, for example: :file:`/dev/ttyACM0`
+   * *baud_rate* - Specifies the baud rate to use.
+   The Thread Co-Processor sample supports baud rate ``1000000``.
 
-     'spinel+hdlc+uart://\ *ncp_uart_device*\ ?uart-reset'
+For example, to use ``ot-daemon``, enter the following commands::
```

Comment:
```suggestion
For example, to use ``ot-daemon``, enter the following commands:
```
Only need the one colon since you're also manually opening the code block.